ios/FluentUI/Drawer/DrawerShadowView.swift

```diff
@@ -5,7 +5,9 @@
 
 import UIKit
 
-class DrawerShadowView: UIView {
+class DrawerShadowView: UIView, Shadowable {
+    var ambientShadow: CALayer?
+    var keyShadow: CALayer?
```

Comment:
Could you test shadow overrides? Looks like the overrides we have right now are just for the background color, but we should make sure that the other ones are working, too.

Reply:
Yep just tested and shadow overrides work. Do you want me to add the changes as part of this PR?

Reply:
It'd probably be good to add them to make it easier to test in the future.